### PR TITLE
capitalize class calls

### PR DESCRIPTION
**Pull request by vzuburlis**

### CODE REVIEW

File: src/core/classes/TableSchema.php [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<?php
```
Comment (panigrc) [inline src/core/classes/TableSchema.php:1]: these changes could be made in another pull request

File: src/core/controllers/login.php [hunk 2]
```diff
@@ -112,6 +112,26 @@ function password_resetAction()
       new sendmail(['email'=>$email, 'subject'=>$subject, 'message'=>$msg, 'headers'=>$headers]);
     }
 
-    view::includeFile('login-change-emailed.php');
+    View::includeFile('login-change-emailed.php');
+  }
+
+  function authAction()
```
Comment (panigrc) [inline src/core/controllers/login.php:118]: is this a new feature ?

File: src/core/models/post.php [hunk 3]
```diff
@@ -10,7 +10,7 @@ static function getById($id)
 
   static function getByIdSlug($id)
   {
-    $db = \gila::slaveDB();
+    $db = \Gila::slaveDB();
```
Comment (panigrc) [inline src/core/models/post.php:13]: You could do `use Gila` and remove the namespace definition.

File: src/core/package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name":"Gila CMS",
-  "version":"1.12.4",
+  "version":"1.12.5",
```
Comment (panigrc) [inline src/core/package.json:3]: This could cause conflicts if another hotfix is merged before this.

File: tests/phpunit/class-gTable.php [hunk 5]
```diff
@@ -34,4 +35,21 @@ public function test_gTable()
     $row = $gtable->getRow(['id'=>$rows[0]['id']], ['select'=>['id','title']]);
     $this->assertEquals('Post Tile', $row['title']);
   }
+
+  public function test_where()
```
Comment (panigrc) [inline tests/phpunit/class-gTable.php:39]: new feature?

Comment (vzuburlis) [reply tests/phpunit/class-gTable.php:39]: new test

File: src/core/package.json (hunk 4 above)
Comment (vzuburlis) [reply src/core/package.json:3]: Not a problem, after release cersion shall be updated

File: src/core/models/post.php (hunk 3 above)
Comment (vzuburlis) [reply src/core/models/post.php:13]: That's cool

File: src/core/classes/TableSchema.php (hunk 1 above)
Comment (vzuburlis) [reply src/core/classes/TableSchema.php:1]: nai tha kanw allo pr :)
